components/03-modules/gallery/gallery.hbs

```diff
@@ -0,0 +1,86 @@
+<div class="
+         gallery {{ class }}
+         {{#if typeHorizontal }}
+             gallery--horizontal
+         {{else}}
+             gallery--vertical
+         {{/if}}
+     "
+     {{{ attributes }}}
+>
+    <div class="
+             gallery__stage
+             {{ stage.class }}
+             {{#if typeHorizontal }}
+                 gallery__stage--horizontal
+             {{else}}
+                 gallery__stage--vertical
+             {{/if}}
+         "
+    >
+        <div class="gallery__arr gallery__arr--prev">
+        </div>
+        <div class="gallery__shaft">
+            <div class="gallery__frame">
+                {{#if productLabel }}
+                <span class="
+                          gallery__product-label
+                          gallery__product--{{productLabel.type}}
+                      "
+                >
+                    {{ productLabel.text }}
+                </span>
+                {{/if}}
+                {{ render '@image' imageMain }}
+            </div>
+        </div>
+        <div class="gallery__arr gallery__arr--next">
+        </div>
+    </div>
+    <div class="gallery__nav
+                {{#if typeHorizontal }}
+                    gallery__nav--horizontal
+                {{else}}
+                    gallery__nav--vertical
+                {{/if}}
+                {{ nav.class }}
+         "
+   >
+        <div class="gallery__thumb-arr">
```

Comment:
in Magento this will be just div or `a` tag or `button`?

Reply:
answer below